front/src/styles/scss/applications/operationalStudies/_scenario.scss

```
@@ -66,79 +109,108 @@
-
-        .scenario-details-modify-button {
-          background-color: transparent;
-          border: none;
-          transition: 0.2s;
-          padding: 0.25rem 0.5rem;
-          font-size: 1rem;
-          border-radius: 0.25rem;
+      }
+    }
 
-          .scenario-details-modify-button-text {
-            transition: opacity 0.2s;
-            opacity: 0;
-            font-size: 0.9rem;
-            margin-right: 0.5rem;
-          }
+    .scenario-details-name {
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      font-weight: 600;
+      font-size: 1.5rem;
+      color: var(--black);
+      font-size: 1.125rem;
+      text-align: left;
+      line-height: 1.5rem;
+      margin-top: 0.375rem;
+    }
 
-          &:hover {
-            background-color: var(--secondary);
-            color: var(--white);
+    .scenario-description {
+      position: relative;
+      text-wrap: wrap;
 
-            .scenario-details-modify-button-text {
-              opacity: 1;
-            }
-          }
-        }
+      .scenario-details-description {
+        color: var(--grey50);
+        font-size: 14px;
+        font-weight: 400;
+        line-height: 1.25rem;
+        margin-top: 0.25rem;
+        word-break: break-all;
       }
 
-      .scenario-details-infra-name {
-        display: flex;
-        align-items: center;
-
-        img {
-          height: 1rem;
-        }
+      .scenario-collapse-button {
+        position: absolute;
+        padding-bottom: 0.188rem;
+        padding-left: 0.125rem;
+        height: 2.125rem;
+        width: 1.5rem;
+        color: var(--grey40);
+        background-color: var(--ambiantB10);
+        border-radius: 0.313rem 0 0 0.313rem;
+        top: 0;
+        transform: translateX(1.5rem);
       }
 
-      .scenario-details-infra-error {
-        color: var(--red);
-        font-weight: 500;
-        font-size: 0.9rem;
+      .update-scenario {
+        display: none;
+        position: absolute;
+        right: 2.5rem;
+        bottom: -3rem;
+        padding: 0.188rem 0.5rem;
+        transition: 0.2s;
+        height: 2rem;
+        width: 2rem;
+        border-radius: 0.25rem;
+        box-shadow: 0 0 0 0.094rem rgba(255, 255, 255, 1) inset;
+        opacity: 1;
+        background-color: var(--ambiantB10);
+        color: var(--primary60);
+        &:hover {
+          background-color: var(--white);
+          color: var(--primary80);
+        }
       }
-
-      .scenario-details-description {
-        overflow: auto;
-        max-height: 4rem;
-        font-size: 0.9rem;
-        white-space: break-spaces;
+    }
+    .scenario-details-electrical-profile-set {
+      font-size: 0.75rem;
+      font-weight: 400;
+      line-height: 1rem;
+      margin-top: 0.375rem;
+      color: var(--black);
+    }
+    .scenario-details-infra-name {
+      font-size: 0.75rem;
+      font-weight: 400;
+      line-height: 1rem;
+      color: var(--black);
```

For the moment we will drop the old design system, we should not use the old var but create new ones (same for white above)

```suggestion
      color: var(--black100);
    }
    .scenario-details-infra-name {
      font-size: 0.75rem;
      font-weight: 400;
      line-height: 1rem;
      color: var(--black100);
```

---

Done